Source/ORTS.Common/Conversions.cs

```diff
@@ -720,6 +720,25 @@ public static string FormatLargeMass(float massKg, bool isMetric, bool isUK)
                 return FormatMass(massKg, isMetric);
         }
 
+        public static string FormatMassBar( float massKg)
+        {
+            string massBar;
+            var range = (int) Math.Ceiling(massKg / 20000);
+            switch (range)
+            {
+                case 1: massBar = "*"; break;
+                case 2: massBar = "**"; break;
+                case 3: massBar = "***"; break;
+                case 4: massBar = "****"; break;
+                case 5: massBar = "*****"; break;
+                case 6: massBar = "******"; break;
+                case 7: massBar = "*******"; break;
+                case 8: massBar = "********"; break;
+                default: massBar = massKg < 1.0 ? "?" : "********>"; break;
+            }
+            return massBar;
+        }
```

Comment:
I don't think this is appropriate in this shared area, as it's not a generic conversion that could have use anywhere else; let's move it into the ContentManager project.

This should also be simplified with the knowledge that `new string('*', count)` will give a string containing `count` `*` characters.

Reply:
Moved to ContentInfo.cs (it already had a Format function). Also simplified as suggested.